src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -1685,9 +1684,11 @@ public static Builder builder() {
    * @throws DockerCertificateException if we could not build a DockerCertificates object
    */
   public static Builder fromEnv() throws DockerCertificateException {
-    final String endpoint = fromNullable(getenv("DOCKER_HOST")).or(defaultEndpoint());
+    final DockerUtils dockerUtils = new DockerUtils();
+    final String endpoint = fromNullable(getenv("DOCKER_HOST"))
```

Comment:
This exact series of method calls—`fromNullable(getenv("DOCKER_HOST")).or(dockerUtils.defaultDockerEndpoint());`—is used multiple times in different places. It seems to me this would be better as a single method in `DockerUtils`, something like `getHost()`.

Same with `fromNullable(getenv("DOCKER_CERT_PATH")).or(dockerUtils.defaultCertPath())`.